lxd/network/openvswitch/ovn.go

```diff
@@ -2047,3 +2082,154 @@ func (o *OVN) LogicalRouterPeeringDelete(opts OVNRouterPeering) error {
 
 	return nil
 }
+
+// GetHardwareAddress gets the mac address of the ovn managed network.
+func (o *OVN) GetHardwareAddress(chassisGroup OVNChassisGroup) (string, error) {
```

Comment:
This function should take an `OVNRouterPort` as an argument rather than a `OVNChassisGroup`, it may be used to access the MAC address of any logical router port, not one specific to a chassis group. Lets not pass LXD concepts into this package if we can help it.

Reply:
The `ovn` driver in LXD has a `getRouterIntPortName` function that will return the logical router port for the internal network, that can be passed in to this function.